s4e-backend/src/main/resources/db/migration/V41__add_invitations_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE invitation (
+    id BIGSERIAL PRIMARY KEY,
+    email VARCHAR NOT NULL,
+    created_at TIMESTAMP,
+    created_by VARCHAR,
+    last_modified_at TIMESTAMP,
+    last_modified_by VARCHAR,
+    token VARCHAR UNIQUE NOT NULL,
+    status VARCHAR NOT NULL,
+    institution_id BIGSERIAL REFERENCES institution,
```

Comment:
I know in `V31__add_favourite_product_app_users_table.sql` this is done with `BIGSERIAL`, however, this is erroneous. It creates a new sequence which will populate the column, only this default is overriden by hibernate and correct reference is persisted to db. This is a tricky behaviour, which leads to problems. Instead, do it as for example in `V16__add_saved_view_table.sql`, in this case it would be:
```sql
institution_id BIGINT NOT NULL REFERENCES institution ON DELETE CASCADE
```
This ensures the reference is set and exists. Moreover, if the institution is deleted all invitations are removed too.